macros/schema_tests/aggregate_functions/expect_column_stdev_to_be_greater_than.sql

```diff
@@ -0,0 +1,42 @@
+{% macro test_expect_column_stdev_to_be_greater_than(model, column_name,
+                                                    greater_than_value,
```

Comment:
We've been just using `value` as the parameter name in other macros (I think that came from the GE API). Probably good for now to try to keep these consistent.

Reply:
Got it.